Libplanet/PoS/DelegationInfo.cs

```diff
@@ -0,0 +1,52 @@
+using System;
+using Bencodex.Types;
+
+namespace Libplanet.PoS
+{
+    [Serializable]
+    public class DelegationInfo
```

Comment:
Is `XXXInfo` mean inner data of 'XXX'? 

IMO If the place with the subject of data does not use a name that represents the data, and the place that uses the data takes the name that represents the data, I think ambiguity about the name and behavior arises.

In this case, my suggestion is `Delegation` to `Delegate`, `DelegationInfo` to `Delegation`. Because the current `Delegation` class is a behavior class. so the verb is way more representative of this feature. not noun.

Reply:
Additionally, this review is not limited to this but applies to most classes on a similar level.

Reply:
> Is XXXInfo mean inner data of 'XXX'?

=> Right.
> In this case, my suggestion is Delegation to Delegate, DelegationInfo to Delegation. Because the current Delegation class is a behavior class. so the verb is way more representative of this feature. not noun.

=> Okay, I'll reflect your suggestion.